frontend/src/scenes/toolbar-launch/AuthorizedUrlsTable.scss

```diff
@@ -0,0 +1,21 @@
+@import '~/vars';
+
+.authorized-urls-table {
+    .authorized-url-col {
```

Comment:
The checkmark icon is a bit off center currently, I'd suggest `align-items: center; display: flex;` here